spec/pg_lock_spec.rb

```diff
@@ -14,6 +14,13 @@
     expect_output_has_message(out: out, count: 1, msg: "PgLock::UnableToLockError")
   end
 
+  it "lock! can be called without raising an error" do
+    key = testing_key("lock! does not raise an error")
+    out = run("env PG_LOCK_KEY='#{key}' SLEEP_FOR=0 bundle exec ruby #{PgLockSpawn.fixture_path("lock!.rb")}")
```

Comment:
@schneems I think this isn't failing because the block in [lock.rb!](https://github.com/heroku/pg_lock/blob/master/spec/fixtures/lock!.rb) returns `3`. Changing it to return `nil` or `false` will raise a `PgLock::UnableToLockError` exception in version `0.2.0`.